README.md

```diff
@@ -25,7 +25,7 @@ Both apps have the exact same set of features, so you might find that simply ope
 
 ### Installing on macOS
 
-If you're using the desktop app on macOS, when you run it you will likely be greeted with some error about the app not being able to be opened, or not verified by Apple, or something similar. The easiest way to get around this is to run `xattr -cr` on the file you downloaded. So let's say you downloaded `appium-inspector.dmg`. Then you would run `xattr -cr appium-inspector.dmg` before opening the disk image. The same goes for the zip version (or the .app itself).
+If you're using the desktop app on macOS, when you run it you will likely be greeted with some error about the app not being able to be opened, or not verified by Apple, or something similar. The easiest way to get around this is to run `xattr -cr` on the file you downloaded. So let's say you downloaded `appium-inspector.dmg` and copy `Appium Inspector.app` in `/Applications` inside the disk image. Then you would run `xattr -cr "/Applications/Appium Inspector.app"` before opening it. The same goes for the zip version (or the .app itself).
```

Comment:
Perhaps do these lines need update? These changes are already in main branch.

Reply:
this is done in the latest commit, thank you .